CMSIS/RTOS2/FreeRTOS/Source/cmsis_os2.c

```diff
@@ -1301,6 +1299,7 @@ osTimerId_t osTimerNew (osTimerFunc_t func, osTimerType_t type, void *argument,
       }
       else {
         mem = 0;
+        name = NULL;
```

Comment:
Please move this back to its original location (line 1284).
I know `name` is assigned two times when attributes are provided but the whole module tends to initialize variables somewhere at the top and modifies them later after some decision making. It just makes things simple and consistent - and we can survive extra CPU instructions when building without optimization.

Reply:
Done